CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 
 ## Master (unreleased)
 
+### Breaking Changes
+
+* Removed `type` param from `status_tag` and some CSS classes [#4989][] by [@javierjulio][]
+  * The method signature has changed from:
+    ```ruby
+    status_tag(status, :ok, class: 'completed', label: 'on')
+    # now changes to:
+    status_tag(status, class: 'completed', label: 'on')
+    ```
+  * The following CSS classes have been **removed**:
+    ```css
+    .status_tag {
+      &.ok, &.published, &.complete, &.completed, &.green { background: #8daa92; }
+      &.warn, &.warning, &.orange { background: #e29b20; }
+      &.error, &.errored, &.red { background: #d45f53; }
+    }
+    ```
```

Comment:
This is no longer a breaking change (not yet). I guess we should move it to a "deprecations" section or something like that.